### PR TITLE
Quadratic test 2.2

### PR DESCRIPTION
**Pull request by Abdullahjavednesar**

### CODE REVIEW

**Comment by Abdullahjavednesar** [comment]
@parsoyaarihant I have minimized test cases.

**Comment by arihantparsoya** [comment]
Thanks, better to have only one test/expression type. I will do it.

File: sympy/integrals/rubi/tests/test_quadratic.py [hunk 1]
```diff
@@ -0,0 +1,820 @@
...
+
+# Integrands of the form (b x+c x**S(2))**p with p symbolic
+[(b*x+c*x**S(2))**p,x,S(1),-(-c*x/b)**(-S(1)-p)*(b*x+c*x**S(2))**(S(1)+p)*hypergeom([-p,S(1)+p],[S(2)+p],(b+c*x)/b)/(b*(S(1)+p))],
+
+# Integrands of the form (a+b x+c x**S(2))**p when b**S(2)-S(4) a c=S(0)
+
+# Integrands of the form (a**S(2)+S(2) a b x+b**S(2) x**S(2))**(p/S(2))
+
+[sqrt(S(4)+S(12)*x+S(9)*x**S(2)),x,S(1),S(1)/S(6)*(S(2)+S(3)*x)*sqrt(S(4)+S(12)*x+S(9)*x**S(2))],
+[S(1)/sqrt(-S(4)+S(12)*x-S(9)*x**S(2)),x,S(2),-S(1)/S(3)*(S(2)-S(3)*x)*log(S(2)-S(3)*x)/sqrt(-S(4)+S(12)*x-S(9)*x**S(2))],
```
Comment (arihantparsoya) [inline sympy/integrals/rubi/tests/test_quadratic.py:30]: these tests are different expresison types right?

**Comment by arihantparsoya** [comment]
I cannot merge the test file as it is. I will test it and merge.

**Pull request closed without merging**